docs/src/app/components/Divider/ExampleList.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import Divider from 'material-ui/lib/divider';
+import List from 'material-ui/lib/lists/list';
+import ListItem from 'material-ui/lib/lists/list-item';
+
+import MobileTearSheet from '../mobile-tear-sheet';
+
+const DividerExampleList = () => (
```

Comment:
It's alright for the examples, but don't follow it in the code. or we'll have ugly diffs :grin: